src/VisualStudio/CSharp/Impl/Debugging/LocationInfoGetter.cs

```diff
@@ -8,33 +10,16 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Editor.Implementation.Debugging;
 using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.VisualStudio.LanguageServices.Implementation.Debugging;
+using Roslyn.Utilities;
 
 namespace Microsoft.VisualStudio.LanguageServices.CSharp.Debugging
 {
     internal static class LocationInfoGetter
     {
-        private static readonly SymbolDisplayFormat s_nameFormat =
-            new SymbolDisplayFormat(
-                globalNamespaceStyle: SymbolDisplayGlobalNamespaceStyle.Omitted,
-                typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces,
-                propertyStyle: SymbolDisplayPropertyStyle.NameOnly,
-                genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters | SymbolDisplayGenericsOptions.IncludeVariance,
-                memberOptions: SymbolDisplayMemberOptions.IncludeContainingType | SymbolDisplayMemberOptions.IncludeParameters | SymbolDisplayMemberOptions.IncludeExplicitInterface,
-                parameterOptions:
-                    SymbolDisplayParameterOptions.IncludeParamsRefOut |
-                    SymbolDisplayParameterOptions.IncludeExtensionThis |
-                    SymbolDisplayParameterOptions.IncludeType |
-                    SymbolDisplayParameterOptions.IncludeName,
-                miscellaneousOptions:
-                    SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
-                    SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
+        private static readonly ObjectPool<StringBuilder> s_stringBuilderPool = SharedPools.Default<StringBuilder>();
 
         internal static async Task<DebugLocationInfo> GetInfoAsync(Document document, int position, CancellationToken cancellationToken)
         {
-            string name = null;
-            int lineOffset = 0;
-
             // Note that we get the current partial solution here.  Technically, this means that we may
             // not fully understand the signature of the member.  But that's ok.  We just need this
             // symbol so we can create a display string to put into the debugger.  If we try to
```

Comment:
Upon closer inspection, it would appear that it's not even accurate for the previous code.  It's describing exactly the problem that was manifesting as if it should not occur.  I'll fix it.